vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,344 +1,233 @@
-implementing a set of interfaces on the server side. The backend is a
-Spring Boot microservice that provides a pluggable interface allowing
-for the injection of any underlying process or automation toolkit. The
-interfaces and steps for creating a new PDA backend implementation are
-provided below. This document provides an overview of the Entando
-Process Driven Apps (PDA) plugin. The initial section covers the
-business functionality and user facing micro frontends. The second
-section covers the architecture and the points of extension for the PDA
-plugin. The third provides a detailed technical overview for developers
-to add new functionality to a PDA implementation.
+# Entando Process Driven Applications Plugin
 
-# Terminology
+## Introduction
 
-## Micro Frontend
+The Entando Process Driven Applications (PDA) plugin is engineered to provide a rich and full-featured user experience while facilitating the management and completion of business processes and automation.  The solution template includes: 
 
-A Micro frontend architecture is an approach to developing web
-application as a composition of small frontend apps. Instead of writing
-a large monolith frontend application, the application is broken down
-into domain specific micro frontends, which are self-contained and can
-be developed and deployed independently.
+- A general purpose UX layer created from micro frontends and can be implemented for any business process or task engine. The UX layer can serve data via the included Entando integration adapter or by implementing a set of interfaces on the server side.
+- A Spring Boot microservice backend providing a pluggable interface for the injection of underlying processes or automation toolkits. The interfaces and steps for creating a new PDA backend implementation are described in further detail [here](link).
 
-## Process Automation Manager (PAM)
+This tutorial will guide you through installing the PDA plugin using the Entando Component Repository (ECR) and an Entando bundle. The goal of this exercise is to demonstrate the scope of process automation functionality enabled by integrating the following components: 
 
-A business process automation engine built and maintained by Red Hat
+- custom Micro Frontends
+- Process Automation Manager
+- Backend for Frontend
 
-## Business Process Modeling Notation (bpmn)
+Key elements of the template are reviewed in the [Application Details section](#application-details) below.
 
-Business Process Model and Notation (BPMN) is a graphical representation
-for specifying business processes in a business process model.
+## Installation
 
-## Decision Model and Notation (dmn)
+There are numerous assets installed as part of the Entando PDA plugin. Entando Bundles can include more or less components, depending on objectives. It is recommended that organizations develop guidelines for bundle sizing that fit the goals of their applications and teams.
```

Comment:
"Entando bundles", lowercase?

Reply:
ack! i thought if something was Entando-specific it was capitalized?? like an Entando Bundle vs any old bundle? @nshaw and @jyunmitch

Reply:
@Lyd1aCla1r3 Maybe only when we include Entando in front of it but it's a general technique so I'd definitely not capitalize when we simply refer to bundle or bundles.